rasa/nlu/utils/__init__.py

```diff
@@ -6,13 +6,6 @@
 import rasa.shared.utils.io
 
```

Comment:
FWIW none of the remaining utils in the file seem NLU specific

Reply:
yep, I'd say it's out of the scope of this PR though.